tgui/packages/tgui/interfaces/RapidPipeDispenser.js

```diff
@@ -40,18 +40,19 @@ const TOOLS = [
 
 export const ColorItem = (props, context) => {
   const { act, data } = useBackend(context);
-  const { space } = props;
   const { selected_color } = data;
   return (
     <LabeledList.Item label="Color">
-      {space ? <span>&nbsp;</span> : ''}
-      <Box inline width="64px" color={data.paint_colors[selected_color]}>
-        {capitalizeAll(selected_color)}
-      </Box>
       {Object.keys(data.paint_colors).map((colorName) => (
```

Comment:
paint colors should be destructured and the object.keys array be made into a const prior to the jsx. lack of any type of optional chaining is making me a little nervous